docker/mainnet/gh-attestation/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e  # Exit on error
+
+# Ensure required environment variables are set
+if [[ -z "$BUNDLE_PATH" || -z "$TRUSTED_ROOT_PATH" ]]; then
+  echo "❌ ERROR: BUNDLE_PATH and TRUSTED_ROOT_PATH environment variables must be set."
+  exit 1
+fi
+
+# Define the image and repo (since they are fixed)
+IMAGE="index.docker.io/blockstack/sbtc:$TAG"  # You can pass $TAG as environment variable
+REPO="stacks-network/sbtc"
+
+# Verifying attestation
+echo "✅ Verifying attestation for image: $IMAGE..."
+apt-get update
+apt-get install -y curl
```

Comment:
Should we pin `curl`? Although I got bit by pinning it (and distro removing the package) in the past. Or move the install to the image so that it's baked into it